src/mechanics.rs

```diff
@@ -55,24 +55,18 @@ pub enum EntryType {
 /// Describes the optionality mode of the `author` and `editor` fields.
 #[derive(Clone, Debug)]
 pub enum AuthorMode {
+    /// Neither of the fields are required to be set.
+    NoneRequired,
+    /// At least one of the fields must be present.
+    OneRequired,
```

Comment:
That's way clearer than `AuthorEditorRequired`.

Reply:
I'm still not super happy with the enum, but I didn't find a better solution and it's not part of the public API anymore.